Bottom: 9004c781f11b8440c1c0e28eb17aa17f83e72ecc
Top:    821d9efd730bdf2dce61c9c049639a5a8de226d8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:13 +0000

Fix undo in MoveObjectAndRubberband

XXX: Need to check if the path which previously missed
     IncrementUndoSerialNumber() was doing so for a reason.


---

diff --git a/src/move.c b/src/move.c
index cfdf881..15cbddc 100644
--- a/src/move.c
+++ b/src/move.c
@@ -946,6 +946,7 @@ MoveObjectAndRubberband (int Type, void *Ptr1, void *Ptr2, void *Ptr3,
       MoveLinePoint (Ptr1, moving_line, &moving_line->Point2);
 
       ptr2 = moving_line;
+      IncrementUndoSerialNumber ();
     }
   else
     {
